src/Components/ProductsList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import "./Products.css"
+
+function ProductsList(props) {
+
+    const {productData, handleClick} = props
+
+    const itemList = productData.map((item, i) =>{
+        return(
+
+            <li key={i}>
+                <h2>{item.name}</h2>
+                <p>Price: ${item.price.toFixed(2)}</p>
+                <button onClick={handleClick}>Add To Cart</button>
```

Comment:
Each button you create will need to add a different item to the shopping cart. How can you design a click handler that is flexible enough to work for each button?

The most common approach to this problem is to write handleClick so that it accepts the item to add to the shopping cart as an argument. Then you may create the click handler like this: onClick={() => handleClick(item)}